package.json

```diff
@@ -47,15 +48,9 @@
     ]
   },
   "lint-staged": {
-    "*.{js, jsx, css, json}": [
-      "yarn run lint:fix",
-      "pretty-quick --staged",
-      "git add"
-    ]
+    "*.{js, jsx, css, json}": []
```

Comment:
Please re-enable linting for code formatting, this keep out source code consistent and easy to maintain